vault/provider_test.go

```diff
@@ -513,3 +513,137 @@ func TestAccProviderToken(t *testing.T) {
 		})
 	}
 }
+
+func TestAccTokenName(t *testing.T) {
+
+	// unset VAULT_TOKEN_NAME
+	defer func() {
+		if err := os.Unsetenv("VAULT_TOKEN_NAME"); err != nil {
```

Comment:
I learned a few months ago that we don't actually need to do this. Turns out that all os env variables set by a test get wiped out before the next test starts, and they don't apply to tests in parallel either.

Reply:
Thanks for your explanation. I removed this.